Makefile

```diff
@@ -42,9 +42,10 @@ build-docker-push: build-docker
 chart:
 	mkdir -p  $(ROOT_DIR)/build
 	cp -rf $(ROOT_DIR)/chart $(ROOT_DIR)/build/chart
-	sed -i -e 's/tag:.*/tag: '${TAG}'/' $(ROOT_DIR)/build/chart/values.yaml
-	sed -i -e 's|repository:.*|repository: '${REPO}'|' $(ROOT_DIR)/build/chart/values.yaml
-	helm package --version ${CHART_VERSION} --app-version ${GIT_TAG} -d $(ROOT_DIR)/build/ $(ROOT_DIR)/build/chart
+	sed -i -e 's/%OPERATOR_VERSION%/${CHART_VERSION}/' $(ROOT_DIR)/build/chart/values.yaml
```

Comment:
I dont think this is correct, tag needs to be the tag of the image built and we currently push the images with the vTAG-COMMIT format.

Reply:
for some odd reason the comment I made here disappeared... 🤷🏽‍♂️ Anyway, this follows the current OBS setup. I am fine using any convention, however I think it would be good to use the same in OBS and here, we just need to define the criteria.